service/src/main/resources/changelog/changesets/20230918_remove_method_prepopulation.yaml

```diff
@@ -0,0 +1,57 @@
+databaseChangeLog:
+  - changeSet:
+      id: "remove fetch_sra_to_bam workflow"
+      author: scottdil
+      preConditions:
+        - onFail: MARK_RAN # Skips over the changeset but mark it as executed.
+        - sqlCheck:
+            expectedResult: 1 # Only delete if it hasn't been run before (template is only runset)
```

Comment:
We want to keep the method if it has already been run in the workspace. Otherwise, we can safely delete. This may make importing the workflows as a group slightly more painful... But this is also primarily a just-in-case for backwards compatibility.